demodoctrine/src/Controller/Admin/QuotesController.php

```diff
@@ -0,0 +1,252 @@
+<?php
+/**
+ * 2007-2020 PrestaShop
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Academic Free License 3.0 (AFL-3.0).
+ * It is also available through the world-wide-web at this URL: https://opensource.org/licenses/AFL-3.0
+ */
+
```

Comment:
Don't forget `declare(strict_types=1);` on new files.
See https://github.com/PrestaShop/docs/pull/518

Reply:
Actually this module is targeting 176, so it is still compatible with PHP 5.6, I think I'll remove the strict typing everywhere just in case

Reply:
So the core fix PR has been rescoped for 177, so the compatibility of this module also evolves ^^ I've added the strict type declaration everywhere, I still need to add strict typing in prototypes where needed